src/check/canonicalize/CIR.zig

```diff
@@ -1075,15 +1090,15 @@ pub fn toSexprTypesStr(ir: *CIR, writer: std.io.AnyWriter, maybe_expr_idx: ?Expr
                 .assign => |assign_pat| {
                     var def_node = SExpr.init(gpa, "patt");
 
-                    def_node.appendRegion(gpa, ir.calcRegionInfo(assign_pat.region));
+                    ir.appendRegionInfoToSexprNodeFromRegion(&def_node, assign_pat.region);
 
                     // Get the type variable for this definition
                     // Each definition has a type_var at its node index which represents the type of the definition
-                    const def_var = try ir.idxToTypeVar(&ir.env.types, def_idx);
+                    const def_var = ir.idxToTypeVar(&ir.env.types, def_idx) catch |err| exitOnOom(err);
 
```

Comment:
We have been slowly moving back to using `try` instead of `exitOnOom` everywhere.